test.md

```diff
@@ -0,0 +1,172 @@
+---
```

Comment:
Another create a LinuxVM doc, that's 4 now. Shouldn't be in the root and we shouldn't have multiple that, at least from the title, appear the same.